developer/docs/help/reference/file-types/kps.md

```diff
@@ -17,11 +17,156 @@ Details:
 Distributed with keyboard:
 :   No. This is a development file and should not be distributed.
 
-## File format
+# File format
 
+## Schema
+
+The schema for .kps is documented at:
+https://github.com/keymanapp/keyman/tree/master/common/schemas/kps
+
+## Strings
 The optional `<Strings>` section of the file can be included to
 customise the text in the bootstrap installer. The default strings are
 found in the file:
 [strings.xml](https://github.com/keymanapp/keyman/blob/stable-14.0/windows/src/desktop/setup/locale/en/strings.xml).
 You can add your own strings for a given package which are used when
 compiling as a bundled installer.
+
+
+## File.Source
+
+The `Name` element may be either a local filename, or a GitHub raw permanent
+URL, conforming to the pattern:
+
+```
+<Name>https://github.com/{owner}/{repo}/raw/{commit}/[path/]{filename}</Name>
+```
+
+We will add a `Source` element to .kps `Package.Files.File`. We would support
+several sources:
+* local - local file system
+* flo - fonts.languagetechnology.org
+* github
+* noto - in the future
+
+## local: `<Source>` element is omitted
```

Comment:
This took me a few re-reads of the part about "`<Source>` element is omitted."
That phrase doesn't appear in the other source headings (flo, github, noto).

Is this conveying for local sources, the `<Source>` element will **always** be omitted? 
vs other sources the `<Source>` element can be omitted?